src/components/ListPeople/ListPeople.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+interface Props {
+  people: Person[]
+  query: string,
+  onQuery?: (event: React.ChangeEvent<HTMLInputElement>) => void,
+  onSelected?: (person: Person) => void
+}
+
+export const ListPeople: React.FC<Props> = React.memo(
```

Comment:
You made the correct separation, but this component needs a refactor

1. `Autocomplete` or `PeopleAutocomplete` is a better name for the component
2. Please implement all properties, as required by the task
3. What is the reason for storing `query` outside the component? You have the input inside, so you should incapsulate this state inside as well